### PR TITLE
PrestaShop 8.1.7 release notes

### PR DESCRIPTION
**Pull request by kpodemski**

### CODE REVIEW

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+### Notable changes in version 8.1.7
+
+Check out some of the most notable improvements and bug fixes from this version.
+
+**Bug fixes**
+
+There are a few bug fixes in this version that are worth mentioning.
+
+- [@Progi1984](https://github.com/Progi1984) backported a fix from [@unlocomqx](https://github.com/unlocomqx) to fix the issue with the quantity check when there are customized products in cart. ([#36342](https://github.com/PrestaShop/PrestaShop/pull/36342)).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]:
```suggestion
- [@Progi1984](https://github.com/Progi1984) backported a fix from [@unlocomqx](https://github.com/unlocomqx) to fix the issue with the quantity check when there are customized products in a cart. ([#36342](https://github.com/PrestaShop/PrestaShop/pull/36342)).
```

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- [@Hlavtox](https://github.com/Hlavtox) fixed the issue with a specific scenario when multiple carriers are used. ([#36074](https://github.com/PrestaShop/PrestaShop/pull/36074)).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]:
```suggestion
- [@Hlavtox](https://github.com/Hlavtox) fixed the issue where multiple carriers are used and a package does not have an assigned carrier. ([#36074](https://github.com/PrestaShop/PrestaShop/pull/36074)).
```

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- [@rudolfkastl](https://github.com/rudolfkastl) fixed problem with credit slip creation. ([#35861](https://github.com/PrestaShop/PrestaShop/pull/35861)).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]:
```suggestion
- [@rudolfkastl](https://github.com/rudolfkastl) fixed a problem with the credit slip creation. ([#35861](https://github.com/PrestaShop/PrestaShop/pull/35861)).
```

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+You can see the rest of pull requests with bug fixes on [Github](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+milestone%3A8.1.7+label%3A%22Bug+fix%22).
+
+**Improvements**
+
+In version 8.1.7, you will find several improvements. Some of the most notable ones are:
+
+- Thanks to [@webeshop](https://github.com/PrestaShop/PrestaShop/pull/34823), when you duplicate a product, the product date is now correctly set to the current date. ([#34823](https://github.com/PrestaShop/PrestaShop/pull/34823))
+- Export button is now available on the product page. Thank you, [@ChronoBrake](https://github.com/ChronoBrake). ([#35488](https://github.com/PrestaShop/PrestaShop/pull/35488)).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]:
```suggestion
- The export button is now available on the product page. Thank you, [@ChronoBrake](https://github.com/ChronoBrake). ([#35488](https://github.com/PrestaShop/PrestaShop/pull/35488)).
```

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+- [@mattgoud](https://github.com/mattgoud) has improved the product page responsiveness. ([#35775](https://github.com/PrestaShop/PrestaShop/pull/35775)).
+- The wording related to managing pack products has been improved by [@Hlavtox](https://github.com/Hlavtox). ([#35434](https://github.com/PrestaShop/PrestaShop/pull/35434)).
+- Size of fields meta_title and meta_description has been corrected by [@M0rgan01](https://github.com/M0rgan01). ([#35564](https://github.com/PrestaShop/PrestaShop/pull/35564)).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]:
```suggestion
- The field size for meta_title and meta_description has been corrected by [@M0rgan01](https://github.com/M0rgan01). ([#35564](https://github.com/PrestaShop/PrestaShop/pull/35564)).
```

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+- [@jolelievre](https://github.com/jolelievre) fixed miscellaneous bugs on the new product page related to combinations and features management. ([#36244](https://github.com/PrestaShop/PrestaShop/pull/36244)).
+
+Improvements above should make your daily work with managing products easier and more efficient.
+
+Apart from the ones mentioned above, there are other notable improvements in this version. One worth mentioning is the improvement of the performance for cart rules. You can check more details in the [pull request](https://github.com/PrestaShop/PrestaShop/pull/35962) by [@ShaiMagal](https://github.com/ShaiMagal), also, to make life easier for developers, the `EntitySearchInput` component [is now available](https://github.com/PrestaShop/PrestaShop/pull/36306) in the `window.prestashop.component` thanks to [@thomasDelaporte](https://github.com/thomasDelaporte).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]:
```suggestion
Apart from the ones mentioned above, there are other notable improvements in this version. One worth mentioning is the cart rules performance improvement. You can check for more details in the [pull request](https://github.com/PrestaShop/PrestaShop/pull/35962) by [@ShaiMagal](https://github.com/ShaiMagal). Also, to make life easier for developers, the `EntitySearchInput` component [is now available](https://github.com/PrestaShop/PrestaShop/pull/36306) in the `window.prestashop.component` thanks to [@thomasDelaporte](https://github.com/thomasDelaporte).
```

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+Make sure to check the [full list of them](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+milestone%3A8.1.7+label%3ARefactoring%2CImprovement).
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]:
```suggestion
Make sure to check the [full list here](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+milestone%3A8.1.7+label%3ARefactoring%2CImprovement).
```

File: content/news/releases/prestashop-8-1-7-maintenance-release.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+
+Since version 8.1.7 is a "patch" update, upgrading from previous 8.1.x versions should happen without any issues. As for every upgrade, it's highly recommended to do a full manual backup beforehand.
+
+If you encounter any problems during the upgrade that, in your opinion, are not caused by third-party modules or modifications used in your shop, [submit a bug report](https://www.prestashop-project.org/get-involved/report-issues/).
+
+## Acknowledgments
+
+Contributors to this patch version, from project members and individual community contributors, include:
+
+A,B,C
```
Comment (coffrep) [inline content/news/releases/prestashop-8-1-7-maintenance-release.md]: To modify, do let me know if you add people here.

**Review by coffrep — CHANGES_REQUESTED** [review]
Hi @kpodemski thanks for creating and putting together this post for the release! Looks great, I did add a few suggestions. I'm also "requesting changes" as there is the contributor list that might need to be updated or even deleted. Do let me know and I'll approve promptly when you tell me.

**Comment by kpodemski** [comment]
@coffrep I added list of contributors, thanks for the review

**Review by coffrep — APPROVED** [review]
Hi @kpodemski that is great, thanks for adding the contributor list! I'm approving on my end.

**Pull request merged**